webpack.common.config.js

```diff
@@ -9,8 +9,14 @@ const configuration = [];
 
 // Set entries.
 const entries = {
-  feedback_carousel: './eox_nelp/course_experience/frontend/src/components/FeedbackCarousel/index',
-  tenant_stats: './eox_nelp/stats/frontend/src/components/TenantStats/index',
+  tenant_stats: {
+    js: './eox_nelp/stats/frontend/src/components/TenantStats/index',
+    template: 'eox_nelp/stats/templates/tenant_stats.html',
```

Comment:
I thought this was of Webpack entries(template key), but now I know that is your custom implementation. Nice.